src/main/java/org/umssdiplo/automationv01/core/managepage/smartHouse/Header.java

```diff
@@ -0,0 +1,57 @@
+package org.umssdiplo.automationv01.core.managepage.smartHouse;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+
+public class Header  extends BasePage {
+    private SHOrganization shOrganization;
+    private SHEmployee shEmployee;
+    private SHLogin shLogin;
+
+    @FindBy(name = "login_button")
+    private WebElement loginBtn;
+
+    @FindBy(name = "org_button")
+    private WebElement organizationTab;
+
+    @FindBy(name = "emp_button")
+    private WebElement employeeTab;
+
+    public SHOrganization getShOrganization() {
+        if(shOrganization==null){
+            shOrganization= new SHOrganization();
+        }
+        return shOrganization;
+    }
+
+    public SHEmployee getShEmployee() {
+        if(shEmployee==null){
+            shEmployee = new SHEmployee();
+        }
+        return shEmployee;
+    }
+
+    public SHLogin getShLogin() {
+        if(shLogin==null){
+            shLogin= new SHLogin();
+        }
+        return shLogin;
+    }
+
+    public SHLogin clickLoginButton(){
+        CommonEvents.clickButton(loginBtn);
+        return getShLogin();
```

Comment:
you need to build your page object. Please remove this method.
you should use: return new SHLogin();